dashboard/src/components/Config/PkgRepoList/PkgRepoList.tsx

```diff
@@ -87,13 +99,57 @@ function PkgRepoList() {
   }, [allNS, currentNamespace]);
 
   useEffect(() => {
+    const pluginsConfig = {
+      helmGlobalNamespace: helmGlobalNamespace,
+      carvelGlobalNamespace: carvelGlobalNamespace,
+      kubeappsCluster: kubeappsCluster,
+    } as IConfig;
+
+    Promise.all(
+      configuredPlugins
+        .map(plugin =>
+          PackageRepositoriesService.getRepositoriesPermissions(
+            currentNamespace,
+            pluginsConfig,
+            plugin,
+          ),
+        )
+        .filter(i => i !== undefined),
+    ).then(results => {
+      const rbac = new Map<string, IPackageRepositoryPermission>();
+      results.forEach(r => {
+        if (r !== undefined) {
+          rbac.set(JSON.stringify(r.plugin), r);
+        }
+      });
+      setReposRBAC(rbac);
+    });
+
+    // Cluster-wide check
     Kube.canI(cluster, "kubeapps.com", "apprepositories", "list", "")
```

Comment:
unrelated to this PR, but just curious... what this `canSetAllNS` is for?  It seems it's coupled with the helm plugin, no?

Reply:
It was a previously existing flag. It controls if the toggle to show repos from all namespaces is hidden.
![image](https://user-images.githubusercontent.com/67455978/196904274-4749f28f-a8b2-41be-b361-e9c3b6165934.png)
Making this feature cross-plugin could be a task for the new UI #5515.